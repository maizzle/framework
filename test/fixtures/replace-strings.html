<div>test</div>

{head} test {tail}
