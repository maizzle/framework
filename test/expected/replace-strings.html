<div>replace strings test</div>

<%= replace strings test %>
